public/index.html

```diff
@@ -6,15 +6,18 @@
   <title>PSGC (Philippine Standard Geographic Code)</title>
   <link rel="preconnect" href="https://fonts.gstatic.com">
   <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@300;400;500;600;700;800;900&display=swap" rel="stylesheet">
-  <link rel="stylesheet" href="./style.css">
+  <link rel="stylesheet" href="../src/styles/style.css">
+  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/fork-corner/dist/fork-corner.min.css">
+  <script src="https://cdn.jsdelivr.net/npm/fork-corner/dist/fork-corner.min.js" defer></script>
 </head>
 <body>
+  <a href="https://github.com/wareneutron/psgc-api" target="_blank" id="fork-corner"
+    class="fork-corner fc-pos-tr fc-animate-grow fc-theme-github"></a>
   <main>
     <div class="main-content">
       <h1 class='logo'><span id="psgc">psgc</span>.<span id="rest">rest</span></h1>
       <div class="content">
         <h4><a href="https://docs.wareneutron.com/psgc-api"><span>Read the Official Documentation</span></a></h4>
-        <h4><a href="https://github.com/warengonzaga/fork-corner"class='github'>View Github Repository</a></h4>
         <h4>By <a href="#">Wareneutron Developers</a></h4>
```

Comment:
You forgot to add this link "https://github.com/wareneutron" as href value.

Reply:
ok